ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -9,12 +9,12 @@ import {
   getSelectedAccount,
 } from '../../../../selectors';
 import {
-  getMultichainIsEvm,
+  // getMultichainIsEvm,
```

Comment:
This wasn't being used. I've commented it out here, but I wonder if this may be related to the flask build. There is also some code fencing later in this file that also seems flask related.

Reply:
It was. Reference fix: https://github.com/MetaMask/metamask-extension/pull/28576